keyboards/hotdox/hotdox.c

```diff
@@ -0,0 +1,59 @@
+#include "hotdox.h"
+#include "backlight.h"
+#include "quantum.h"
+
+extern inline void ergodox_right_led_1_on(void);
+extern inline void ergodox_right_led_2_on(void);
+extern inline void ergodox_right_led_3_on(void);
+extern inline void ergodox_right_led_on(uint8_t led);
+
+extern inline void ergodox_right_led_1_off(void);
+extern inline void ergodox_right_led_2_off(void);
+extern inline void ergodox_right_led_3_off(void);
+extern inline void ergodox_right_led_off(uint8_t led);
+
+extern inline void ergodox_led_all_off(void);
+
+void ergodox_led_init(void);
+void ergodox_blink_all_leds(void);
+
+void matrix_init_kb(void) {
+  ergodox_led_init();
+  //ergodox_blink_all_leds();
+  matrix_init_user();
+}
+
+void ergodox_led_init(void)
+ {
+     DDRB  |= (1 << PB4);
+     PORTB &= ~(1 << PB4);
+
+     DDRB  |= (1 << PB5);
+     PORTB &= ~(1 << PB5);
+
+     DDRB  |= (1 << PB6);
+     PORTB &= ~(1 << PB6);
+
+     //DDRB  |=  ((1 << PB5) | (1 << PB6));
+     //PORTB &= ~((1 << PB5) | (1 << PB6));
+}
+
+void ergodox_blink_all_leds(void)
+{
+    ergodox_led_all_off();
+    ergodox_led_all_set(LED_BRIGHTNESS_HI);
+    //ergodox_right_led_1_on();
```

Comment:
If all of this is being commented out, you may want to remove all of the delays too. 

And specifically, is there a reason that they are removed/commented out? 
It's fine if they are, but ideally, I'd like to see the behavior kept consistent with the similar boards, if the hardware is there for it. (personal preference)